### ISSUE TITLE
Wrong model update for repeating section if used with pre-filled array

### ISSUE THREAD

**Issue by fdenert** [formly-js/ngx-formly#821]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
Using the new repeating section with an pre-filled array and then adding a new fieldGroup does not update the model correctly.

```
model = { investments: [ {investmentName: 'test'}, {investmentName: 'test'} ] };
```

The updated model looks like that:

```
{ investments: [ {investmentName: 'test'}, {investmentName: 'test'}, {} ] }
```

Only works with empty investments array:
```
model = { investments: [] };
```

**Expected behavior**
The updated model should contain the new investmentName:

```
{ investments: [ {investmentName: 'test'}, {investmentName: 'test'}, {investmentName: 'new value from input'} ] }
```

**Minimal reproduction of the problem with instructions**
https://stackblitz.com/edit/angular-x4bdgg?file=app%2Fapp.component.ts

Add more investments, fill it out, submit and check the result.

(Maybe that's a problem with the Angular version 5.2? It works with version 5.0.x. Or am i missing something with my form setup?)

* **Angular version:** 5.2.8

**Comment by aitboudad** [comment]
fixed in `2.0.0-rc.44`

**Comment by fdenert** [comment]
That was quick.
Thank you for your great work!
